subsys/bluetooth/controller/ll_sw/ull_slave.c

```diff
@@ -348,8 +369,8 @@ void ull_slave_done(struct node_rx_event_done *done, uint32_t *ticks_drift_plus,
 	}
 }
 
-void ull_slave_ticker_cb(uint32_t ticks_at_expire, uint32_t remainder, uint16_t lazy,
-			 void *param)
+void ull_slave_ticker_cb(uint32_t ticks_at_expire, uint32_t remainder,
+	uint16_t lazy, void *param)
```

Comment:
Use leading tabs, and then spaces to align with parameters.

Reply:
(See previous comment.)